package.json

```diff
@@ -32,7 +34,7 @@
     "ts-node": "^3.2.0",
     "tslint": "^5.7.0",
     "typedoc": "^0.8",
-    "typescript": "^2.7.1",
+    "typescript": "next",
```

Comment:
I'm just curious as to why this change? I might have missed a discussion on this, feel free to disregard my comment if so :)

Reply:
This change will be erased shortly.

Reply:
It's reverted now.